.github/workflows/build_docker_image.yml

```diff
@@ -0,0 +1,59 @@
+name: Build Docker Image
+
+on:
+  push:
+    branches:
+      - master
+      #- main
+      #- 'v*.*.*'
+  workflow_dispatch:
+  schedule:
+    - cron: '00 4 01 * *'  # At 04:00 on day-of-month 01.
```

Comment:
I don't think this cron serves a useful purpose.

Reply:
Since "scratch" is used as the base image, you're right. Otherwise, I like to use it to ensure that at least operating system updates are regularly integrated into the image.